packages/extension-koni-ui/src/Popup/Home/index.tsx

```diff
@@ -464,7 +464,7 @@ export default React.memo(styled(Wrapper)(({ theme }: WrapperProps) => `
 
   .home-tab-contents {
     flex: 1;
-    overflow: auto;
+    overflow: overlay;
```

Comment:
"overlay" does not work on Firefox. Please find another solution.
![image](https://user-images.githubusercontent.com/92499846/161703449-c530da5b-4dc7-4983-8afb-fff6f41d8675.png)